twitter_api/main.py

```diff
@@ -17,6 +21,164 @@ def before_request():
 
 # implement your views here
 
+def _md5_hash(pw):
+    return md5(pw.encode('utf-8')).hexdigest()
+
+
+@app.route('/login', methods=['POST'])
+@has_json_keys('password', 'username', error = 400)
+def login():
+    #Accept json with username and pw
+    #return acces token
+    user = request.json['username']
+    received_password = request.json['password']
```

Comment:
believe it or not mind blown, didn't find out about this .json method until it was too late >_<